### PR TITLE
added StakeHolder-Upload-Multi-Part-Document

### PR DESCRIPTION
**Pull request by madhankumar3103**
i have added stakeholder upload multi part documents

### CODE REVIEW

**Review by Rrathinasabapath — CHANGES_REQUESTED** [review]
Please make the changes.

File: src/Hyperwallet/Hyperwallet.php [hunk 1]
```diff
@@ -2079,5 +2080,104 @@ public function listVenmoAccountStatusTransitions($userToken, $venmoAccountToken
             return new VenmoAccountStatusTransition($entry);
         });
     }
+    /**
+     * Upload documents for user endpoint
```
Comment (arao6) [inline src/Hyperwallet/Hyperwallet.php:2084]: Please change the name of the model to BusinessStakeholder

File: src/Hyperwallet/Hyperwallet.php [hunk 2]
```diff
@@ -2079,5 +2080,104 @@ public function listVenmoAccountStatusTransitions($userToken, $venmoAccountToken
...
+        if (empty($businessToken)) {
+            throw new HyperwalletArgumentException('businessToken is required!');
+        }
+        $body = $this->client->doPut('/rest/v4/users/{user-token}/business-stakeholders/{business-stakeholders}', array('user-token' => $userToken,'business-stakeholders' => $businessToken), $businessUser, array());
+        return new BusinessUser($body);
+    }
+
+    /**
+     * List all Businessusers
+     *
```
Comment (arao6) [inline src/Hyperwallet/Hyperwallet.php:2166]: Pls update Businessusers to BusinessStakeholders

**Review by Rrathinasabapath — CHANGES_REQUESTED** [review]
Pls fix the one comment.

File: src/Hyperwallet/Hyperwallet.php [hunk 3]
```diff
@@ -2079,5 +2080,104 @@ public function listVenmoAccountStatusTransitions($userToken, $venmoAccountToken
...
+     * @throws HyperwalletApiException
+     */
+    public function uploadDocumentsForBusinessStakeholder($userToken, $businessToken, $options) {
+        if (empty($userToken)) {
+            throw new HyperwalletArgumentException('userToken is required!');
+        }
+        if (empty($businessToken)) {
+            throw new HyperwalletArgumentException('businessToken is required!');
+        }
+        $body = $this->client->putMultipartData('/rest/v4/users/{user-token}/business-stakeholders/{business-stakeholders}', array('user-token' => $userToken,'business-stakeholders' => $businessToken), $options);
```
Comment (ramahalingam) [inline src/Hyperwallet/Hyperwallet.php:2120]: {business-stakeholders} can be renamed.

**Review by arao6 — APPROVED** [review]

**Review by Rrathinasabapath — APPROVED** [review]
Reviewed the changes. Ready to be reviewed by Peter.

**Review by vkorotchenko — APPROVED** [review]

**Review by ramahalingam — APPROVED** [review]

**Pull request merged**